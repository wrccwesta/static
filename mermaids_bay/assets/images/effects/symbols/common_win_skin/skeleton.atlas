
skeleton.png
size: 2355,372
format: RGBA8888
filter: Linear,Linear
repeat: none
10
  rotate: true
  xy: 960, 76
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
9
  rotate: true
  xy: 681, 76
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
a
  rotate: true
  xy: 1239, 76
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
fx_bubble_one
  rotate: true
  xy: 395, 2
  size: 74, 73
  orig: 74, 73
  offset: 0, 0
  index: -1
fx_mist
  rotate: true
  xy: 2, 6
  size: 364, 391
  orig: 364, 391
  offset: 0, 0
  index: -1
fx_water
  rotate: true
  xy: 395, 78
  size: 292, 284
  orig: 292, 284
  offset: 0, 0
  index: -1
j
  rotate: true
  xy: 1518, 76
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
k
  rotate: true
  xy: 1797, 76
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
q
  rotate: true
  xy: 2076, 76
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
