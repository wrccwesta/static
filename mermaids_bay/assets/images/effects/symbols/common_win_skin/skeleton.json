{"skeleton":{"hash":"483sh70XWsqKXqrw14d88wCnVSs","spine":"3.7.19-beta","width":1922.34,"height":1084.16,"images":"../spine/","audio":"D:/SHARED/SLOT new/new/mermaids/anim"},"bones":[{"name":"root"},{"name":"b1","parent":"root","length":55.23,"rotation":90,"x":147,"y":-127.52},{"name":"b2","parent":"root","length":55.23,"rotation":90,"x":147,"y":-127.52},{"name":"bone","parent":"root","length":49.04,"rotation":87.66,"x":250.5,"y":-205.5},{"name":"bubble one 1","parent":"root","length":26.78,"rotation":88.03,"x":245.81,"y":-129.85},{"name":"bubble one 2","parent":"root","length":26.78,"rotation":88.03,"x":206.81,"y":-189.85,"scaleX":0.526,"scaleY":0.526},{"name":"bubble one 3","parent":"root","length":26.78,"rotation":88.03,"x":262.81,"y":-213.85,"scaleY":-1},{"name":"bubble one 4","parent":"root","length":26.78,"rotation":88.03,"x":300.81,"y":-140.85,"scaleX":0.312,"scaleY":0.312},{"name":"bubble one 5","parent":"root","length":26.78,"rotation":-41.41,"x":182.81,"y":-82.85,"scaleX":0.526,"scaleY":0.526},{"name":"bubble one 6","parent":"root","length":26.78,"rotation":88.03,"x":244.81,"y":-178.85,"scaleX":-1,"scaleY":-1},{"name":"bubble one 7","parent":"root","length":26.78,"rotation":88.03,"x":281.81,"y":-186.85,"scaleX":0.312,"scaleY":0.312},{"name":"bubble one 8","parent":"root","length":26.78,"rotation":88.03,"x":231.81,"y":-164.85,"scaleX":0.312,"scaleY":0.312},{"name":"bubble one 9","parent":"root","length":26.78,"rotation":88.03,"x":76.81,"y":-152.85},{"name":"bubble one 10","parent":"root","length":26.78,"rotation":88.03,"x":16.81,"y":-140.85,"scaleX":0.526,"scaleY":0.526},{"name":"bubble one 11","parent":"root","length":26.78,"rotation":88.03,"x":49.81,"y":-206.85,"scaleY":-1},{"name":"bubble one 12","parent":"root","length":26.78,"rotation":88.03,"x":19.81,"y":-188.85,"scaleX":0.312,"scaleY":0.312},{"name":"bubble one 13","parent":"root","length":26.78,"rotation":-41.41,"x":182.81,"y":-82.85,"scaleX":0.526,"scaleY":0.526},{"name":"bubble one 14","parent":"root","length":26.78,"rotation":88.03,"x":45.81,"y":-111.85,"scaleX":-1,"scaleY":-1},{"name":"bubble one 15","parent":"root","length":26.78,"rotation":88.03,"x":-17.19,"y":-114.85,"scaleX":0.312,"scaleY":0.312},{"name":"mist","parent":"root","length":37.01,"rotation":88.45,"x":153,"y":-119.5},{"name":"symb","parent":"root","length":43.25,"rotation":90,"x":147,"y":-138.5},{"name":"water","parent":"root","length":30.07,"rotation":90,"x":147,"y":-138.5}],"slots":[{"name":"bg","bone":"root"},{"name":"fx_mist","bone":"mist","attachment":"fx_mist","blend":"additive"},{"name":"symb","bone":"symb","attachment":"symb"},{"name":"fx_water","bone":"water","attachment":"fx_water","blend":"additive"},{"name":"fx_bubble_one","bone":"bubble one 1","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one9","bone":"bubble one 9","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one3","bone":"bubble one 3","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one11","bone":"bubble one 11","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one6","bone":"bubble one 6","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one14","bone":"bubble one 14","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one2","bone":"bubble one 2","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one10","bone":"bubble one 10","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one5","bone":"bubble one 5","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one13","bone":"bubble one 13","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one4","bone":"bubble one 4","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one12","bone":"bubble one 12","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one8","bone":"bubble one 8","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one7","bone":"bubble one 7","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one15","bone":"bubble one 15","attachment":"fx_bubble_one","blend":"additive"}],"skins":{"default":{"fx_bubble_one":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one10":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one11":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one12":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one13":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one14":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one15":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one2":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one3":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one4":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one5":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one6":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one7":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one8":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_bubble_one9":{"fx_bubble_one":{"x":10.12,"y":1.27,"rotation":-88.03,"width":74,"height":73}},"fx_mist":{"fx_mist":{"x":-9.31,"y":11.25,"rotation":3.65,"width":364,"height":391}},"fx_water":{"fx_water":{"x":20,"y":3,"rotation":-90,"width":292,"height":284}}},"9":{"symb":{"symb":{"name":"9","rotation":-90,"width":294,"height":277}}},"10":{"symb":{"symb":{"name":"10","rotation":-90,"width":294,"height":277}}},"a":{"symb":{"symb":{"name":"a","rotation":-90,"width":294,"height":277}}},"j":{"symb":{"symb":{"name":"j","rotation":-90,"width":294,"height":277}}},"k":{"symb":{"symb":{"name":"k","rotation":-90,"width":294,"height":277}}},"q":{"symb":{"symb":{"name":"q","rotation":-90,"width":294,"height":277}}}},"animations":{"win":{"slots":{"fx_bubble_one":{"color":[{"time":0,"color":"ffffff00"},{"time":0.5,"color":"ffffffff"},{"time":1,"color":"ffffff00"}]},"fx_bubble_one2":{"color":[{"time":0,"color":"ffffff00"},{"time":0.3333,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]},"fx_bubble_one3":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.6,"color":"ffffff00"},{"time":1.1,"color":"ffffffff"},{"time":1.6,"color":"ffffff00"}]},"fx_bubble_one4":{"color":[{"time":0,"color":"ffffff00"},{"time":0.3333,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]},"fx_bubble_one5":{"color":[{"time":0,"color":"ffffff00"},{"time":0.3333,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]},"fx_bubble_one6":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.3667,"color":"ffffff00"},{"time":0.8667,"color":"ffffffff"},{"time":1.3667,"color":"ffffff00"}]},"fx_bubble_one7":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.5667,"color":"ffffff00"},{"time":0.9,"color":"ffffffff"},{"time":1.2333,"color":"ffffff00"}]},"fx_bubble_one8":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.8333,"color":"ffffff00"},{"time":1.3,"color":"ffffffff"},{"time":1.7667,"color":"ffffff00"}]},"fx_bubble_one9":{"color":[{"time":0,"color":"ffffff00"},{"time":0.5,"color":"ffffffff"},{"time":1,"color":"ffffff00"}]},"fx_bubble_one10":{"color":[{"time":0,"color":"ffffff00"},{"time":0.3333,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]},"fx_bubble_one11":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.6,"color":"ffffff00"},{"time":1.1,"color":"ffffffff"},{"time":1.6,"color":"ffffff00"}]},"fx_bubble_one12":{"color":[{"time":0,"color":"ffffff00"},{"time":0.3333,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]},"fx_bubble_one13":{"color":[{"time":0,"color":"ffffff00"},{"time":0.3333,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]},"fx_bubble_one14":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.3667,"color":"ffffff00"},{"time":0.8667,"color":"ffffffff"},{"time":1.3667,"color":"ffffff00"}]},"fx_bubble_one15":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.8333,"color":"ffffff00"},{"time":1.3333,"color":"ffffffff"},{"time":1.8,"color":"ffffff00"}]},"fx_mist":{"color":[{"time":0,"color":"ffffff00"},{"time":0.8333,"color":"ffffffff"},{"time":1.8,"color":"ffffff00"}]},"fx_water":{"color":[{"time":0,"color":"ffffff00","curve":[0,0.8,0.468,0.7]},{"time":0.2667,"color":"ffffffff"},{"time":0.6667,"color":"ffffff00"}]}},"bones":{"mist":{"rotate":[{"time":0,"angle":0},{"time":1.8,"angle":-9.8}],"translate":[{"time":0,"x":-1,"y":0},{"time":0.2667,"x":0,"y":-16},{"time":0.6667,"x":0,"y":-5}],"scale":[{"time":0,"x":0.5,"y":0.5},{"time":1.8,"x":0.76,"y":0.76}]},"water":{"translate":[{"time":0,"x":-1,"y":74,"curve":[0,0.8,0.468,0.7]},{"time":0.2667,"x":0,"y":-16},{"time":0.6667,"x":0,"y":-5}],"scale":[{"time":0,"x":0.2,"y":0.2,"curve":[0,0.8,0.468,0.7]},{"time":0.2667,"x":0.8,"y":0.75},{"time":0.6667,"x":0.9,"y":0.9}]},"symb":{"rotate":[{"time":0,"angle":0},{"time":0.9,"angle":1.4},{"time":1.8,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.2667,"x":0,"y":-20},{"time":0.8333,"x":0,"y":9},{"time":1.8,"x":0,"y":0}]},"b1":{"translate":[{"time":0,"x":2,"y":-40},{"time":0.6667,"x":-1,"y":37}],"scale":[{"time":0,"x":1,"y":1},{"time":0.6667,"x":1.1,"y":0.9}]},"b2":{"translate":[{"time":0,"x":0.5,"y":-1.5},{"time":0.3333,"x":-1,"y":37},{"time":0.3667,"x":2,"y":-40},{"time":0.9,"x":0.5,"y":-1.5}],"scale":[{"time":0,"x":0.95,"y":0.95},{"time":0.3333,"x":0.9,"y":0.9},{"time":0.3667,"x":1,"y":1},{"time":0.9,"x":1.05,"y":0.95}]},"bubble one 1":{"rotate":[{"time":0,"angle":0},{"time":1,"angle":-13.57}],"translate":[{"time":0,"x":-34,"y":-23},{"time":1,"x":-26,"y":104}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":1,"x":1,"y":1}]},"bubble one 2":{"rotate":[{"time":0,"angle":0},{"time":0.6667,"angle":-13.57}],"translate":[{"time":0,"x":-17,"y":-17},{"time":0.6667,"x":-17,"y":79}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":0.6667,"x":1,"y":1}]},"bubble one 3":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.6,"angle":0},{"time":1.6,"angle":16.72}],"translate":[{"time":0,"x":-22,"y":35,"curve":"stepped"},{"time":0.6,"x":-22,"y":35},{"time":1.6,"x":-31,"y":129}],"scale":[{"time":0,"x":0.936,"y":0.936},{"time":0.6,"x":0.729,"y":0.729},{"time":1.6,"x":1.01,"y":1.01}]},"bubble one 4":{"rotate":[{"time":0,"angle":0},{"time":0.6667,"angle":-13.57}],"translate":[{"time":0,"x":-62,"y":5},{"time":0.6667,"x":-68,"y":109}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":0.6667,"x":1,"y":1}]},"bubble one 5":{"rotate":[{"time":0,"angle":0},{"time":0.6667,"angle":-103.68}],"translate":[{"time":0,"x":-17,"y":-17},{"time":0.6667,"x":-3,"y":82}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":0.6667,"x":1,"y":1}]},"bubble one 6":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.3667,"angle":0},{"time":1.3667,"angle":-59.3}],"translate":[{"time":0,"x":-22,"y":35},{"time":0.3667,"x":-10,"y":31},{"time":1.3667,"x":13,"y":151}],"scale":[{"time":0,"x":0.936,"y":0.936,"curve":"stepped"},{"time":0.3667,"x":0.936,"y":0.936},{"time":1.3667,"x":1,"y":1}]},"bubble one 7":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.5667,"angle":0},{"time":1.2333,"angle":-13.57}],"translate":[{"time":0,"x":-62,"y":5,"curve":"stepped"},{"time":0.5667,"x":-62,"y":5},{"time":1.2333,"x":-68,"y":109}],"scale":[{"time":0,"x":0.737,"y":0.737,"curve":"stepped"},{"time":0.5667,"x":0.737,"y":0.737},{"time":1.2333,"x":1,"y":1}]},"bubble one 8":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0},{"time":1.7667,"angle":-13.57}],"translate":[{"time":0,"x":-62,"y":5},{"time":0.8333,"x":-12,"y":-10},{"time":1.7667,"x":-13,"y":101.58}],"scale":[{"time":0,"x":0.737,"y":0.737,"curve":"stepped"},{"time":0.8333,"x":0.737,"y":0.737},{"time":1.7667,"x":1,"y":1}]},"bubble one 9":{"rotate":[{"time":0,"angle":0},{"time":1,"angle":-13.57}],"translate":[{"time":0,"x":-1,"y":-34},{"time":1,"x":6,"y":108}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":1,"x":1,"y":1}]},"bubble one 10":{"rotate":[{"time":0,"angle":0},{"time":0.6667,"angle":-13.57}],"translate":[{"time":0,"x":20,"y":-20},{"time":0.6667,"x":40,"y":62}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":0.6667,"x":1,"y":1}]},"bubble one 11":{"rotate":[{"time":0,"angle":16.09,"curve":"stepped"},{"time":0.6,"angle":16.09},{"time":1.6,"angle":-82.9}],"translate":[{"time":0,"x":0,"y":14},{"time":0.6,"x":-1.39,"y":-4.03},{"time":1.6,"x":2.61,"y":81.65}],"scale":[{"time":0,"x":0.729,"y":0.729},{"time":0.6,"x":0.519,"y":0.519},{"time":1.6,"x":0.942,"y":0.942}]},"bubble one 12":{"rotate":[{"time":0,"angle":0},{"time":0.6667,"angle":-13.57}],"translate":[{"time":0,"x":9,"y":24},{"time":0.6667,"x":15,"y":150}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":0.6667,"x":1,"y":1}]},"bubble one 13":{"rotate":[{"time":0,"angle":-98.64},{"time":0.6667,"angle":14.36}],"translate":[{"time":0,"x":-128,"y":-118},{"time":0.6667,"x":-123,"y":33}],"scale":[{"time":0,"x":0.737,"y":0.737},{"time":0.6667,"x":1,"y":1}]},"bubble one 14":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.3667,"angle":0},{"time":1.3667,"angle":-59.3}],"translate":[{"time":0,"x":-22,"y":35},{"time":0.3667,"x":-8,"y":-28},{"time":1.3667,"x":38,"y":96}],"scale":[{"time":0,"x":0.936,"y":0.936},{"time":0.3667,"x":0.764,"y":0.764},{"time":1.3667,"x":0.881,"y":0.881}]},"bubble one 15":{"rotate":[{"time":0,"angle":0,"curve":"stepped"},{"time":0.8333,"angle":0},{"time":1.8,"angle":33.39}],"translate":[{"time":0,"x":87,"y":-73,"curve":"stepped"},{"time":0.8333,"x":87,"y":-73},{"time":1.8,"x":101.8,"y":79.61}],"scale":[{"time":0,"x":0.737,"y":0.737,"curve":"stepped"},{"time":0.8333,"x":0.737,"y":0.737},{"time":1.8,"x":1,"y":1}]}}}}}