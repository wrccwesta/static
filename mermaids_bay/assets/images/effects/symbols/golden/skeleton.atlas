
skeleton.png
size: 1405,374
format: RGBA8888
filter: Linear,Linear
repeat: none
crab
  rotate: true
  xy: 395, 78
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
fish
  rotate: true
  xy: 674, 78
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
fx_bubble_one
  rotate: true
  xy: 395, 2
  size: 74, 73
  orig: 74, 73
  offset: 0, 0
  index: -1
fx_mist
  rotate: true
  xy: 2, 8
  size: 364, 391
  orig: 364, 391
  offset: 0, 0
  index: -1
fx_spark_yellow
  rotate: false
  xy: 1232, 202
  size: 171, 170
  orig: 171, 170
  offset: 0, 0
  index: -1
horse
  rotate: true
  xy: 953, 78
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
