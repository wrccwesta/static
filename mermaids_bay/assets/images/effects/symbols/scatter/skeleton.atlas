
skeleton.png
size: 912,368
format: RGBA8888
filter: Linear,Linear
repeat: none
fx_bubble_one
  rotate: true
  xy: 721, 17
  size: 74, 73
  orig: 74, 73
  offset: 0, 0
  index: -1
fx_mist
  rotate: true
  xy: 2, 2
  size: 364, 391
  orig: 364, 391
  offset: 0, 0
  index: -1
fx_spark_yellow
  rotate: false
  xy: 674, 196
  size: 171, 170
  orig: 171, 170
  offset: 0, 0
  index: -1
fx_star_white
  rotate: true
  xy: 655, 4
  size: 66, 64
  orig: 66, 64
  offset: 0, 0
  index: -1
scatter
  rotate: true
  xy: 395, 72
  size: 294, 277
  orig: 294, 277
  offset: 0, 0
  index: -1
scatter_freespins
  rotate: true
  xy: 847, 93
  size: 273, 63
  orig: 273, 63
  offset: 0, 0
  index: -1
scatter_text
  rotate: false
  xy: 395, 2
  size: 258, 68
  orig: 258, 68
  offset: 0, 0
  index: -1
