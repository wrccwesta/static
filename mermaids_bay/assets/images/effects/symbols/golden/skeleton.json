{"skeleton":{"hash":"K7eqOFE5OSCX9qCs7NdSyBWa7eM","spine":"3.7.19-beta","width":1920,"height":1080,"images":"../spine/","audio":""},"bones":[{"name":"root"},{"name":"body","parent":"root","length":29.6,"rotation":90,"x":147,"y":-138.5},{"name":"bubble 1","parent":"root","length":43.1,"rotation":91.4,"x":245.46,"y":-171.74},{"name":"bubble 2","parent":"root","length":43.1,"rotation":91.4,"x":64.68,"y":-140.21},{"name":"bubble 3","parent":"root","length":43.1,"rotation":91.4,"x":223.39,"y":-125.5,"scaleX":0.408,"scaleY":-0.408},{"name":"bubble 4","parent":"root","length":43.1,"rotation":91.4,"x":84.65,"y":-194.87,"scaleX":0.408,"scaleY":-0.408},{"name":"bubble 5","parent":"root","length":43.1,"rotation":91.4,"x":266.48,"y":-149.67,"scaleX":0.258,"scaleY":-0.258},{"name":"mist","parent":"root","length":45.19,"rotation":90,"x":147.85,"y":-142.55},{"name":"spark","parent":"body","length":33.65},{"name":"spark2","parent":"body","length":33.65,"x":56.75,"y":23.12,"scaleX":0.606,"scaleY":0.606}],"slots":[{"name":"bg","bone":"root"},{"name":"fx_mist","bone":"mist","attachment":"fx_mist","blend":"additive"},{"name":"horse","bone":"body","attachment":"sym"},{"name":"fx_spark_yellow","bone":"spark","attachment":"fx_spark_yellow","blend":"additive"},{"name":"fx_spark_yellow2","bone":"spark2","attachment":"fx_spark_yellow","blend":"additive"},{"name":"fx_bubble_one","bone":"bubble 1","color":"27a39aff","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one3","bone":"bubble 3","color":"27a39aff","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one5","bone":"bubble 5","color":"27a39aff","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one4","bone":"bubble 4","color":"27a39aff","attachment":"fx_bubble_one","blend":"additive"},{"name":"fx_bubble_one2","bone":"bubble 2","color":"27a39aff","attachment":"fx_bubble_one","blend":"additive"}],"skins":{"default":{"fx_bubble_one":{"fx_bubble_one":{"x":-1.05,"y":0.03,"rotation":-91.4,"width":74,"height":73}},"fx_bubble_one2":{"fx_bubble_one":{"x":-1.05,"y":0.03,"rotation":-91.4,"width":74,"height":73}},"fx_bubble_one3":{"fx_bubble_one":{"x":-1.05,"y":0.03,"rotation":-91.4,"width":74,"height":73}},"fx_bubble_one4":{"fx_bubble_one":{"x":-1.05,"y":0.03,"rotation":-91.4,"width":74,"height":73}},"fx_bubble_one5":{"fx_bubble_one":{"x":-1.05,"y":0.03,"rotation":-91.4,"width":74,"height":73}},"fx_mist":{"fx_mist":{"x":1.05,"y":-3.15,"scaleX":0.719,"scaleY":0.719,"rotation":-90,"width":364,"height":391}},"fx_spark_yellow":{"fx_spark_yellow":{"width":171,"height":170}},"fx_spark_yellow2":{"fx_spark_yellow":{"width":171,"height":170}}},"crab":{"horse":{"sym":{"name":"crab","rotation":-90,"width":294,"height":277}}},"fishes":{"horse":{"sym":{"name":"fish","rotation":-90,"width":294,"height":277}}},"horse":{"horse":{"sym":{"name":"horse","rotation":-90,"width":294,"height":277}}}},"animations":{"win":{"slots":{"fx_bubble_one":{"color":[{"time":0,"color":"26a39900","curve":"stepped"},{"time":0.1667,"color":"26a39900"},{"time":0.7333,"color":"25a399ff"},{"time":1.3333,"color":"26a39900"}]},"fx_bubble_one2":{"color":[{"time":0,"color":"26a39900"},{"time":0.5667,"color":"25a399ff"},{"time":1.1667,"color":"26a39900"}]},"fx_bubble_one3":{"color":[{"time":0,"color":"26a39900"},{"time":0.5667,"color":"25a399ff"},{"time":1.1667,"color":"26a39900"}]},"fx_bubble_one4":{"color":[{"time":0,"color":"26a39900","curve":"stepped"},{"time":0.1667,"color":"26a39900"},{"time":0.7333,"color":"25a399ff"},{"time":1.3333,"color":"26a39900"}]},"fx_bubble_one5":{"color":[{"time":0,"color":"26a39900","curve":"stepped"},{"time":0.5667,"color":"26a39900"},{"time":1.1333,"color":"25a399ff"},{"time":1.7333,"color":"26a39900"}]},"fx_mist":{"color":[{"time":0,"color":"ffffff00"},{"time":0.9,"color":"ffffffff"},{"time":1.8,"color":"ffffff00"}]},"fx_spark_yellow":{"color":[{"time":0,"color":"ffffff00"},{"time":0.6667,"color":"ffffffff"},{"time":1.3333,"color":"ffffff00"}]},"fx_spark_yellow2":{"color":[{"time":0,"color":"ffffff00","curve":"stepped"},{"time":0.8333,"color":"ffffff00"},{"time":1.3333,"color":"ffffffff"},{"time":1.8,"color":"ffffff00"}]}},"bones":{"body":{"rotate":[{"time":0,"angle":0},{"time":0.8333,"angle":1.22},{"time":1.8,"angle":0}],"translate":[{"time":0,"x":0,"y":0},{"time":0.3333,"x":0,"y":-11.56},{"time":1.0667,"x":0,"y":5.78},{"time":1.8,"x":0,"y":0}]},"spark":{"rotate":[{"time":0,"angle":0},{"time":1.3333,"angle":-48.66}]},"spark2":{"rotate":[{"time":0.8333,"angle":0},{"time":1.8,"angle":-48.66}]},"bubble 1":{"rotate":[{"time":0.1667,"angle":0},{"time":1.3333,"angle":-32.81}],"translate":[{"time":0.1667,"x":0,"y":0},{"time":1.3333,"x":-10.51,"y":97.74}],"scale":[{"time":0,"x":0.632,"y":0.632,"curve":"stepped"},{"time":0.1667,"x":0.632,"y":0.632},{"time":1.3333,"x":1,"y":1}]},"bubble 2":{"rotate":[{"time":0,"angle":0},{"time":1.1667,"angle":42.06}],"translate":[{"time":0,"x":0,"y":0},{"time":1.1667,"x":-10.51,"y":97.74}],"scale":[{"time":0,"x":0.632,"y":0.632},{"time":1.1667,"x":1,"y":1}]},"bubble 3":{"rotate":[{"time":0,"angle":0},{"time":1.1667,"angle":-32.81}],"translate":[{"time":0,"x":0,"y":0},{"time":1.1667,"x":-10.51,"y":97.74}],"scale":[{"time":0,"x":0.632,"y":0.632},{"time":1.1667,"x":1,"y":1}]},"bubble 4":{"rotate":[{"time":0.1667,"angle":0},{"time":1.3333,"angle":-32.81}],"translate":[{"time":0.1667,"x":0,"y":0},{"time":1.3333,"x":-10.51,"y":97.74}],"scale":[{"time":0,"x":0.632,"y":0.632,"curve":"stepped"},{"time":0.1667,"x":0.632,"y":0.632},{"time":1.3333,"x":1,"y":1}]},"bubble 5":{"rotate":[{"time":0.5667,"angle":0},{"time":1.7333,"angle":-32.81}],"translate":[{"time":0,"x":0,"y":0},{"time":0.5667,"x":-7.36,"y":-32.58},{"time":1.7333,"x":-10.51,"y":97.74}],"scale":[{"time":0,"x":0.632,"y":0.632,"curve":"stepped"},{"time":0.5667,"x":0.632,"y":0.632},{"time":1.7333,"x":1,"y":1}]},"mist":{"rotate":[{"time":0,"angle":0},{"time":1.8,"angle":-10.16}],"scale":[{"time":0,"x":0.842,"y":0.842},{"time":1.8,"x":0.996,"y":0.996}]}}}}}